neqo-transport/src/recovery/mod.rs

```diff
@@ -973,8 +973,9 @@ mod tests {
 
     const ON_SENT_SIZE: usize = 100;
     /// An initial RTT for using with `setup_lr`.
-    const TEST_RTT: Duration = ms(80);
-    const TEST_RTTVAR: Duration = ms(40);
+    const TEST_RTT: Duration = INITIAL_RTT;
+    #[allow(clippy::cast_possible_truncation)]
+    const TEST_RTTVAR: Duration = Duration::from_millis(INITIAL_RTT.as_millis() as u64 / 2);
```

Comment:
I would change these to ms(7000) and ms(3500) instead so that we don't engage the defaults.

Reply:
That change breaks tests:
```
        FAIL [   0.034s] neqo-transport recovery::tests::ack_delay_adjusted
        FAIL [   0.034s] neqo-transport recovery::tests::ack_delay_ignored
        FAIL [   0.036s] neqo-transport recovery::tests::big_gap_loss
        FAIL [   0.035s] neqo-transport recovery::tests::no_new_acks
        FAIL [   0.034s] neqo-transport recovery::tests::reduce_min_rtt
        FAIL [   0.033s] neqo-transport recovery::tests::time_loss_detection_timeout
```